extensions/recents/src/recents.tsx

```diff
@@ -153,90 +149,88 @@ export default function Command(props: { scope?: string | undefined }) {
         ) : null
       }
     >
-      {filtered.map((recent, recentIndex) => (
-        <List.Item
-          key={recentIndex}
-          icon={{ fileIcon: recent.kMDItemPath }}
-          title={recent.kMDItemDisplayName}
-          quickLook={{ path: recent.kMDItemPath, name: recent.kMDItemDisplayName }}
-          actions={
-            <ActionPanel>
-              <Action.Open
-                title={`Open ${recent.kMDItemKind}`}
-                icon={{ fileIcon: recent.kMDItemPath }}
-                target={recent.kMDItemPath}
-                onOpen={() => popToRoot({ clearSearchBar: true })}
-              />
-              <Action.ShowInFinder
-                icon={Icon.Finder}
-                title="Show In Finder"
-                path={recent.kMDItemPath}
-                onShow={() => popToRoot({ clearSearchBar: true })}
-              />
-              <Action.ToggleQuickLook title="Quick Look" shortcut={{ modifiers: ["cmd"], key: "y" }} />
-              <Action.OpenWith
-                title="Open With..."
-                shortcut={{ modifiers: ["cmd"], key: "o" }}
-                path={recent.kMDItemPath}
-                onOpen={() => popToRoot({ clearSearchBar: true })}
-              />
-              <Action
-                title="Show Info in Finder"
-                icon={Icon.Finder}
-                shortcut={{ modifiers: ["cmd"], key: "i" }}
-                onAction={() => showInfoInFinder(recent)}
-              />
-              <ActionPanel.Section>
-                <Action.CopyToClipboard
-                  title={`Copy ${recent.kMDItemKind}`}
-                  shortcut={{ modifiers: ["cmd"], key: "." }}
-                  content={``}
-                  onCopy={() => copyRecentToClipboard(recent)}
+      {filtered.map((recent, recentIndex) => {
+        const { kMDItemPath, kMDItemDisplayName, kMDItemLastUsedDate, kMDItemKind } = recent;
+        const escapedName = kMDItemDisplayName.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+        const regex = new RegExp(`/${escapedName}`);
+        const subtitle = kMDItemPath.replace(userHome, "~").replace(regex, "");
+
+        return (
+          <List.Item
+            key={recentIndex}
+            icon={{ fileIcon: kMDItemPath }}
+            title={kMDItemDisplayName}
+            subtitle={subtitle}
```

Comment:
The diff looks a bit wild due to the added destructuring. Everything in the list aside from the `subtitle` and `accessories` prop remained otherwise untouched.